### PR TITLE
Add base bundle

### PR DESCRIPTION
**Pull request by cedvan**
# ApiExceptionBundle

[Build 
Status](https://travis-ci.org/M6Web/ApiExceptionBundle)
[Total Downloads](https://packagist.org/packages/m6web/api-exception-bundle)
[Latest Stable Version](https://packagist.org/packages/m6web/api-exception-bundle)
[License](https://packagist.org/packages/m6web/api-exception-bundle)

The ApiExceptionBundle manages exceptions for your API and return a clean json.
## Documentation

For documentation, see:

```
Resources/doc/
```
- [Installation](https://github.com/M6Web/ApiExceptionBundle/blob/feature/add-base-bundle/Resources/doc/installation.md)
- [Usage](https://github.com/M6Web/ApiExceptionBundle/blob/feature/add-base-bundle/Resources/doc/usage.md)
- [Variables](https://github.com/M6Web/ApiExceptionBundle/blob/feature/add-base-bundle/Resources/doc/variables.md)
- [Stack Trace](https://github.com/M6Web/ApiExceptionBundle/blob/feature/add-base-bundle/Resources/doc/stack_trace.md)
- [Errors](https://github.com/M6Web/ApiExceptionBundle/blob/feature/add-base-bundle/Resources/doc/errors.md)
- [Form](https://github.com/M6Web/ApiExceptionBundle/blob/feature/add-base-bundle/Resources/doc/form.md)
- [Match](https://github.com/M6Web/ApiExceptionBundle/blob/feature/add-base-bundle/Resources/doc/match.md)
- [Log](https://github.com/M6Web/ApiExceptionBundle/blob/feature/add-base-bundle/Resources/doc/log.md)
- [Configuration](https://github.com/M6Web/ApiExceptionBundle/blob/feature/add-base-bundle/Resources/doc/configuration.md)
- [Unit Tests](https://github.com/M6Web/ApiExceptionBundle/blob/feature/add-base-bundle/Resources/doc/unit_tests.md)
- [About](https://github.com/M6Web/ApiExceptionBundle/blob/feature/add-base-bundle/Resources/doc/about.md)
## Contributing

See [CONTRIBUTING](https://github.com/M6Web/ApiExceptionBundle/blob/feature/add-base-bundle/CONTRIBUTING.md) file.
## Credits

Developped by the Frack Team of M6 Web.
## License

This bundle is released under the MIT license.

### CODE REVIEW

File: DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        $rootNode = $treeBuilder->root('m6web_api_exception');
+
+        $rootNode
+            ->children()
+                ->arrayNode('exception')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->arrayNode('stack_trace')
+                            ->defaultValue([
+                                'dev'     => true,
```
Comment (fabdsp) [inline DependencyInjection/Configuration.php]: Comme on a dit ensemble, plutôt laisser la config par env gérer ces cas.

**Comment by fabdsp** [comment]
:+1:   for a first version

File: Resources/doc/about.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# About
+
+This bundle require **PHP 5.5**
+
+### Dependancies
```
Comment (mikaelrandy) [inline Resources/doc/about.md]: *dependencies

File: Resources/doc/about.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+- [symfony/http-kernel](https://github.com/symfony/http-kernel) : `~2.7`
+- [symfony/dependency-injection](https://github.com/symfony/dependency-injection) : `~2.7`
+- [symfony/monolog-bridge](https://github.com/symfony/monolog-bridge) : `~2.7`
+- [symfony/form](https://github.com/symfony/form) : `~2.7`
+- [symfony/yaml](https://github.com/symfony/yaml) : `~2.7`
+- [symfony/config](https://github.com/symfony/config) : `~2.7`
+- [atoum/atoum](https://github.com/atoum/atoum) : `~2.1`
+- [m6web/coke](https://github.com/M6Web/Coke) : `~2.1`
+- [m6web/symfony2-coding-standard](https://github.com/M6Web/Symfony2-coding-standard) : `~3.1`
```
Comment (mikaelrandy) [inline Resources/doc/about.md]: C'est pas le travail du composer.json ça ?
On ne risque pas d'être toujours en décalage ?

Comment (cedvan) [inline Resources/doc/about.md]: C'était juste pour recenser dans la doc les requires du Bundle, mais le `composer.json` suffit on est d'accord. Je supprime ?

File: Resources/doc/about.md (hunk 2 above)
Comment (cedvan) [inline Resources/doc/about.md]: je corrige

**Comment by mojoLyon** [comment]
:metal:

**Comment by omansour** [comment]
documentation is great :) :clap:

**Comment by cedvan** [comment]
@omansour Thank you :)

**Comment by lnahiro** [comment]
:+1:

**Comment by cedvan** [comment]
New release [1.0.0](https://github.com/M6Web/ApiExceptionBundle/releases/tag/v1.0.0)

**Pull request merged**